base_conditional_image/views/image_view.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
I think it should be `odoo` and not `openerp`

Reply:
It's a V9 branch, so it's ```openerp``` not ```odoo```

Reply:
It's a V9 branch, so it's ```openerp``` not ```odoo```

Reply:
Ok ok, I think I work too much on v10.0 :)